homeassistant/components/ness_alarm/alarm_control_panel.py

```diff
@@ -60,6 +60,7 @@ def __init__(self, client: Client, name: str) -> None:
         """Initialize the alarm panel."""
         self._client = client
         self._attr_name = name
+        self._attr_unique_id = f"{name}-alarm"
```

Comment:
Okay so, name is not a good unique id.

Do we have something truly unique for this device? If not, we should migrate this integration to a config flow so we can use the entry_id